src/utils/common.js

```diff
@@ -65,7 +65,7 @@ const supportedAddressPrefixes = new RegExp(
 
 export const printLog = (...params: any) => {
   if ((isProdEnv && !__DEV__) || isTest) return;
-  console.log(...params);
+  console.log(...params); // eslint-disable-line
```

Comment:
We have a logger util now available for use, if it helps. It's merged into develop. Not a show stopper though.

Reply:
I've added this as this should be the only console.log in the code and when running `lint` check this always was showing as a warning even though we're cool with this.